docs/DeveloperGuide.adoc

```diff
@@ -279,87 +279,362 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
+* is a solo traveller
+* prefers desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: convenient travel assistant for solo travelers who are always on their feet
 
 [appendix]
 == User Stories
 
-Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
+*Priorities*
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+* High (must have) - `* * *`
+* Medium (nice to have) - `* *`
+* Low (not useful) - `*`
+
+[width="80%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a ... |I want to ... |So that I ca ...
+|`* * *` | frequent traveler | add a travel entry with weather, location and time metadata | document my past travels
+
+|`* * *` | traveler always on the go | record short text entries recording my travels, similar to that of a Tweet on Twitter | can keep track of my activities without excessive typing
+
+|`* * *` |user |add an activity into an itinerary list |see my planned activities at a glance and better plan my trip
```

Comment:
May want to specify "user" to a more specific target user (i.e. solo traveler, CLI enthusiast)